jekyll/_cci2/creating-orbs.md

```diff
@@ -331,3 +331,203 @@ To publish an orb, follow the steps listed below as an org Admin.
+
+### Get the new CircleCI CLI
+
+The CircleCI platform enables you to write orbs using the CircleCI CLI. If you choose to work with the CLI, the process of writing orbs will be more efficient because you will be able to use existing CircleCI CLI tools and commands.
+
+#### Installing the CLI for the First Time
+
+If you are installing the new `circleci` CLI for the first time, run the following command:
+
+```
+curl -fLSs https://circle.ci/cli | bash
+```
+
+By default, the `circleci` app will be installed in the `/usr/local/bin directory`. If you do not have write permissions to `/usr/local/bin`, you may need to use the `sudo` command. Alternatively, you may install the CLI to an alternate location by defining the `DESTDIR` environment variable when invoking `bash`:
+
+```
+curl -fLSs https://circle.ci/cli | DESTDIR=/opt/bin/bash
+```
+
+##### Homebrew
+
+If you wish to use Homebrew, run the following command:
+
+```
+brew install circleci
+```
+##### Snapcraft
+
+You may also use Snapcraft to install the CLI by running the following command:
+
+```
+sudo snap install circleci
+```
+
+#### Upgrading From an Existing CLI
+
+If you have used a previous version of the CircleCI and are currently running a version older than `0.1.6`, you will need to run the following commands to upgrade your CLI version to the latest version.
+
+```
+circleci update
+circleci switch
+```
+
+If you do not have write permissions, be sure to use the `sudo` command. This will install the CLI to the `/usr/local/bin` directory.
+
+#### Updating the CircleCI CLI after Installation
+
+The CircleCI comes with a built-in version management system. After you have installed the CLI, you should check to see if there are any updates to the CLI that need to be installed by running the following commands:
+
+```
+circleci update check
+circleci update install
+```
+
+### Configuring the CircleCI CLI
+
+Now that you have installed the CircleCI CLI, you will want to configure the CLI for use. The process for configuring the CLI is simple and straightforward, requiring you only to follow a few steps.
+
+Before you can configure the CLI, you may need to first generate a CircleCI API token from the Personal API Token tab [https://circleci.com/accounts/api]:
+
+```
+$ circleci setup
+```
+
+If you are using the CLI tool on `circleci.com`, make sure to accept the provided default `CircleCI Host`.
+
+If you are a user of a privately installed CircleCI deployment you will have to change the default value to your custom address, for example, circleci.my-org.com.
+
+<aside class="notice">
+CircleCI installed on a private cloud or datacenter does not yet support config processing and orbs; therefore, you will only be able to use `circlecli local execute` (this was previously `circleci build`).
+</aside>
+
+### Validating a Build Config
```

Comment:
This `###` (Validating a Build Config) and the next one (Set the Version Property to 2.1) should get moved out of this section.

Reply:
going to keep these for now